src/audio/SDL_audio.c

```diff
@@ -941,6 +941,9 @@ bool SDL_InitAudio(const char *driver_name)
         driver_name = SDL_GetHint(SDL_HINT_AUDIO_DRIVER);
     }
 
+    // save this off in case we kept any streams from a previous init/quit.
+    SDL_AudioStream *streams = current_audio.existing_streams;
+
```

Comment:
```suggestion
    // save this off in case we kept any streams from a previous init/quit.
    SDL_AudioStream *streams = current_audio.existing_streams;

```